### DISCUSSION TITLE
Alternative to Control-C

### DISCUSSION THREAD

**Discussion by jsampson45** [psoukie/zipchord#224] [Ideas]
Pressing Control-C to copy an item in other software causes Zipchord to pop up a chord entry procedure. I am wondering if Zipchord should use an alternative to Control-C for entering new chords.

**Comment by psoukie** [comment]
That should happen only for a long press of Ctrl+C (I'd like to hear if this happens across all apps or just sometimes.)

But it's also possible to customize the keyboard shortcuts by using ZipChord's tray menu. See under [application keyboard shortcuts]( https://github.com/psoukie/zipchord/wiki/Application-Keyboard-Shortcuts-window)

**Comment by psoukie** [comment]
The application shortcuts became a major pain point once I switched to using physical keys instead of the default Autohotkey implementation that doesn't care which key is pressed but which character it produces.

Autohotkey's streamlined approach is powerful until one needs to support other language layouts which always mattered to me. And at that point, the standard keyboard shortcuts don't play along with the physical keys scan codes.

The result is that in version 2.6, I switched to a command menu triggered by pressing both Shift keys together. So to add shortcuts, the fastest way is to do that and then press A.
